server/src/workers/microservices.ts

```diff
@@ -6,22 +6,19 @@ import { ILoggerRepository } from 'src/interfaces/logger.interface';
 import { WebSocketAdapter } from 'src/middleware/websocket.adapter';
 import { otelSDK } from 'src/utils/instrumentation';
 
-const host = process.env.HOST;
-
 export async function bootstrap() {
   otelSDK.start();
 
-  const port = Number(process.env.MICROSERVICES_PORT) || 3002;
   const app = await NestFactory.create(MicroservicesModule, { bufferLogs: true });
   const logger = await app.resolve(ILoggerRepository);
   logger.setAppName('ImmichMicroservices');
   logger.setContext('ImmichMicroservices');
   app.useLogger(logger);
   app.useWebSocketAdapter(new WebSocketAdapter(app));
 
-  await (host ? app.listen(port, host) : app.listen(port));
+  await app.listen(0);
```

Comment:
we don't actually use this port, so just pick a random one. (TODO: remove listening entirely in the future)